program-list/program-list.json

```diff
@@ -8447,7 +8447,7 @@
     "bug_bounty": true,
     "swag": false,
     "hall_of_fame": true,
-    "safe_harbor": "full"
+    "safe_harbor": "partial"
```

Comment:
```suggestion
    "safe_harbor": "none"
```

Reply:
For this to be considered partial, I think there would need to be at a minimum a commitment to protecting well-intentioned security research from criminal charges under CFAA or DMCA.

The wording being used at time of writing is:

> We will consider activities conducted consistent with this policy to constitute “authorized” conduct and will not pursue civil action or initiate a complaint against you.

This leaves the door wide open to criminal prosecution, in my opinion, and while IANAL this doesn't appear to meet our bar for the definition of even partial safe harbor.

If Voatz were to change this wording I'd definitely reconsider.